doc/source/Getting_started/Troubleshooting.rst

```diff
@@ -122,33 +127,37 @@ variables have been defined.
 
 
 
-Failures in connecting to gRPC API
-----------------------------------
-On Linux, it may happens that PyAEDT fails to initialize a new session of Electronics Desktop
-or to connect to an existing one.
+Failure connecting to the gRPC server
+-------------------------------------
+On Linux, PyAEDT may fail to initialize a new instance of the gRPC server
+or connect to an existing server session.
 This may be due to:
  - Firewall
  - Proxy
  - Permissions
  - License
- - Scheduler used to launch AEDT like LSF
+ - Scheduler (for example if the gRPC server was started from LSF, Slurm, ...)
```

Comment:
```suggestion
 - Scheduler (for example if the gRPC server was started from LSF or Slurm)
```
You will want to add "Slurm" to the accept.text file here: C:\AnsysDev\GitRepos\PyAnsys\pyaedt\doc\styles\Vocab\ANSYS